xml/Windows.Foundation/Size.xml

```diff
@@ -95,9 +97,9 @@
         <Parameter Name="o" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="o">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="o">The <see cref="T:System.Object" /> to compare.</param>
+        <summary>Compares an object to an instance of <see cref="T:Windows.Foundation.Size" /> for equality.</summary>
+        <returns><see langword="true" /> if the sizes are equal; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="o" /> is a <see cref="T:Windows.Foundation.Size" /> value and the sizes of the two objects are equal; otherwise, <see langword="false" />.</returns>
```